docker-compose.yml

```diff
@@ -57,20 +57,26 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:3e478a9
+    image: nebraltd/hm-diag:625ea72
+    depends_on:
+      - dbus-session
     environment:
       - FIRMWARE_VERSION=2021.11.26.1
```

Comment:
Should be [2021.11.22.0-2](https://github.com/NebraLtd/helium-miner-software/pull/277#issuecomment-979873765) or 2021.11.22.0-3. Same for the other FIRMWARE_VERSION.

Reply:
I guess `2021.11.26.1-1` could also work.